scripts/webpack.client.config.js

```diff
@@ -1,88 +1,96 @@
-const webpack = require("webpack");
-const path = require("path");
-const merge = require("webpack-merge");
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const PreloadWebpackPlugin = require("preload-webpack-plugin");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const WebpackPwaManifest = require('webpack-pwa-manifest')
+const webpack = require('webpack');
+const path = require('path');
+const merge = require('webpack-merge');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const PreloadWebpackPlugin = require('preload-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
Super minor, but out of curiosity why the switch to single quotes? I believe we're all using double quotes as per https://github.com/codercom/code-server/blob/master/tslint.json

Reply:
I'm sorry. I had Prettier on again! Will change in a minute.